.github/workflows/build.yml

```
@@ -0,0 +1,39 @@
+        with:
+          version: 10
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: 22.13.1
+
+      - name: Install Dependencies
+        run: pnpm i;
+
+      - name:  Build Affected Projects
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)